tiledb/sm/cpp_api/array.h

```diff
@@ -1528,6 +1528,21 @@ class Array {
     std::memcpy((void*)key->data(), key_c, key_len);
   }
 
+  /*
+   * Retrieve the maximum in memory tile sizes across all fragments in an array.
+   * Encrypted arrays will be supported via config options.
+   *
+   * @param max_in_memory_tile_size The receiver for the max in memory tile
+   * size.
+   */
+  void get_max_in_memory_tile_size(uint64_t* max_in_memory_tile_size) {
```

Comment:
This could return the max_in_memory_tile_size. For CAPI we have to do it this way but the CPP api we can do better.

Reply:
ok. (tho take note that its less consistent with C API and adds additional overhead.  
I'm aware that various of the CPP APIs have similar differences with C APIs - that doesn't mean its a good thing.)